test/check-verify

```diff
@@ -60,8 +60,7 @@ def run(opts, sink):
 
 def main():
     parser = testlib.arg_parser()
-    parser.add_argument('--publish', dest='publish', action='store',
-                        help='Publish results centrally to a sink')
+    parser.add_argument('--publish', action='store', help="Unused")
```

Comment:
Is this really unused? What about https://github.com/cockpit-project/cockpit/pull/3512/files#diff-0867c15b54a65614d0f4c199cac890faR180